### PR TITLE
Add email signup and login

### PR DESCRIPTION
**Pull request by NinaKWelch**
- Updated Facebook sign up & login
- Added email & password login
- Updated Sign Up page
- Included link to terms

### CODE REVIEW

**Review by ilarijn — APPROVED** [review]

File: client/src/services/auth.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import axios from "axios"
+
+const baseUrl = "/api/auth"
+
+export const createNewUser = async (newUser) => {
+  try {
+    const response = await axios.post(`${baseUrl}/email/register`, { user_info: newUser })
```
Comment (ToxPuro) [inline client/src/services/auth.js:7]: No need for user_info, just axios.post(`${baseUrl}/email/register`,  newUser )

File: client/src/services/auth.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    console.log(response.data)
+  } catch (err) {
+    console.log(err.message)
+  }
+}
+
+export const loginUser = async (credentials) => {
+  try {
+    const response = await axios.post(`${baseUrl}/login`, credentials)
```
Comment (ToxPuro) [inline client/src/services/auth.js:16]: If this is email login, it should be  await axios.post(`${baseUrl}/email`, credentials)

Comment (ToxPuro) [reply client/src/services/auth.js:16]: Though I could change it to api/auth/email/login, if that would be more straightforward

**Review by ToxPuro — CHANGES_REQUESTED** [review]
Otherwise from comments looks good!

**Review by ToxPuro — APPROVED** [review]

**Pull request merged**